QEfficient/transformers/models/pytorch_transforms.py

```diff
@@ -385,6 +403,13 @@ class KVCacheTransform(ModuleMappingTransform):
         GemmaDecoderLayer: QEffGemmaDecoderLayer,
         GemmaModel: QEffGemmaModel,
         GemmaForCausalLM: QEffGemmaForCausalLM,
+        # Qwen3Moe
```

Comment:
Also updated ReadMe on what all official models will be supported as part of enabling this arch. thanks